### PR TITLE
214 add features to more button on the student list component

### PR DESCRIPTION
**Pull request by OlhaDorofiychuk**

### CODE REVIEW

File: src/components/Cohorts/StudentList/MoreFeatures.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        setShowMoreFeatures(false)
+      })
+      .catch((err) => console.log(err.response))
+  }
+  const handleChange = (e) => {
+    changedCohortId.cohortId = Number(e.target.value)
+    console.log(changedCohortId)
+    console.log('value', e.target.value)
+    console.log('event on change', e)
+  }
```
Comment (Haroogle) [inline src/components/Cohorts/StudentList/MoreFeatures.js:42]: Remove this console logs and you should be good

File: src/components/Cohorts/StudentList/MoreFeatures.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    client
+      .patch(`/users/${student.id}`, changedCohortId)
+      .then(() => {
+        updateStudentsList()
+        setShowMoreFeatures(false)
+      })
+      .catch((err) => console.log(err.response))
+  }
+  const handleChange = (e) => {
```
Comment (dogezen) [inline src/components/Cohorts/StudentList/MoreFeatures.js:37]: seems like htis function is not doing anything and just console logging things? Please remove (unless you need line 38 to do something, but that instruction won't be updating state...)

**Review by dogezen — CHANGES_REQUESTED** [review]

File: src/components/Cohorts/StudentList/MoreFeatures.js (hunk 2 above)
Comment (OlhaDorofiychuk) [reply src/components/Cohorts/StudentList/MoreFeatures.js:37]: this function updates local variable changedCohortId.cohortId = Number(e.target.value) and then sends it with PATCH request to update state. tested and working

Comment (OlhaDorofiychuk) [reply src/components/Cohorts/StudentList/MoreFeatures.js:37]: Will potentially polish it latter, but it will let us show functionality for now

**Review by dogezen — APPROVED** [review]

**Pull request merged**